guacamole-docker/bin/build-guacamole.sh

```diff
@@ -165,6 +165,21 @@ if [ -f extensions/guacamole-auth-duo/target/*.tar.gz ]; then
         "*.jar"
 fi
 
+#
+# Copy Saml auth extension if it was built
+#
+
+if [ -f extensions/guacamole-auth-saml/target/*.tar.gz ]; then
+    mkdir -p "$DESTINATION/saml"
+    tar -xzf extensions/guacamole-auth-saml/target/*.tar.gz \
+        -C "$DESTINATION/saml/"                             \
+        --wildcards                                        \
+        --no-anchored                                      \
+        --no-wildcards-match-slash                         \
+        --strip-components=1                               \
+        "*.jar"
+fi
+
```

Comment:
I believe this may no longer be necessary due to the restructuring of the SSO extensions (CAS, OpenID, and SAML) into a single .tar.gz as of 1.4.0:

https://github.com/apache/guacamole-client/blob/aa99b4bc8a511a0f96466d16de2c88af9befea83/guacamole-docker/bin/build-guacamole.sh#L113-L122

Can you rebase and recheck against the latest?